node-gui/backend/src/error.rs

```diff
@@ -21,6 +21,8 @@ pub enum BackendError {
     WalletError(String),
     #[error("RPC error: {0}")]
     RpcError(String),
+    #[error("Json error: {0}")]
+    JsonError(String),
```

Comment:
`JsonError` looks too generic, given the context where it's produced.
I'd suggest introducing a more specific error variant that somehow mentions `to_dehexified_json`.

Btw, `RpcError` above seems to be unused.